docs/source/dialogue_representation.rst

```diff
@@ -36,7 +36,7 @@ Annotations
 
 :py:mod:`dialoguekit.core.annotation`
 
-By default, an annotation has a slot and a value, for example, the slot *price* has the value *$10* in the following utterance *"This dish costs $10."*.
```

Comment:
This example could be reused a few lines below for SlotValueAnnotations. There, it could also be mentioned that optionally, the start and end character positions of the value can also be stored.

Reply:
Done.